readme.md

```diff
@@ -125,6 +125,7 @@
 - [Python](https://github.com/vinta/awesome-python)
 	- [Asyncio](https://github.com/timofurrer/awesome-asyncio) - Asynchronous I/O in Python 3.
 	- [Scientific Audio](https://github.com/faroit/awesome-python-scientific-audio) - Scientific research in audio/music.
+	- [Education](https://github.com/quobit/awesome-python-in-education) - Resources about Python in Education.
```

Comment:
The description is not very useful. It just repeats the title.

Reply:
Thank you. I'll write a proper one.